File: contoh.php/nilai.php
<?php

$nama=readline("Masukkan Nama Anda: ");
#nim=readline("Masukkan Nim Anda: ");

$tugas=readline("Masukkan Nilai Tugas: ");
$uts=readline("Masukkan Nilai UTS: ");
$uas=readline("Masukkan Nilai UAS: ");

$nilaiAkhir= ($tugas*0.3) + ($uts*0.3) + ($uas*0.4);

echo "Nilai Akhir= $nilaiAkhir\n";
if($nilaiAkhir>=60)
    echo "Status: LULUS\n";
else
    echo "Status: TIDAK LULUS\n";
?>

File: contoh.java/nilai.java
import java.util.Scanner;

public class nilai {
    public static void main(String[] args) {
        @SuppressWarnings("resource")
        Scanner sc = new Scanner(System.in);
        System.out.println("Masukkan nilai tugas: ");
        double tugas = sc.nextDouble();
        System.out.println("Masukkan nilai uts: ");
        double uts = sc.nextDouble();
        System.out.println("Masukkan nilai uas: ");
        double uas = sc.nextDouble();

        double nilaiAkhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);

        System.out.println("Nilai Akhir: " + nilaiAkhir);
        if(nilaiAkhir >= 60)
            System.out.println("Status: LULUS");
        else
            System.out.println("Status: TIDAK LULUS");
    }
}